app/pages/verificationwithpassword/verificationwithpassword.js

```diff
@@ -81,11 +88,15 @@ export let VerificationWithPassword = React.createClass({
   },
 
   render: function() {
+    if (_.get(this.props, 'notification.message', '') === errorMessages.ERR_BIRTHDAY_MISMATCH) {
+      this.props.trackMetric('VCA Home Verification - Birthday Mismatch')
+    }
```

Comment:
I struggled with this a little. Yes, you're right that it should get out of render (my bad) and into one of the other lifecycle methods. This is a result of the server giving back an error, so we can't track it before the prop comes back. I think that ideally, it would happen where the `verifyCustodialFailure` is created in the `verifyCustodial` async action creator, but that would mean making a direct API call to the metrics tracking, which isn't done anywhere else in the codebase, so I was trying to avoid adding one. Do you think it would be best to add the tracking API call in the action creator?